internal/service/groups/groups.go

```diff
@@ -221,10 +219,9 @@ func (g *groupService) changeGroupOwner(ctx context.Context, group modelsv1.Grou
 
 // DeleteGroup removes all members from the group and sets a deletion timestamp.
 func (g *groupService) DeleteGroup(ctx context.Context, groupID string) error {
-	logger := logging.LoggerFromContext(ctx)
 	validator, err := rbac.NewValidator(ctx)
 	if err != nil {
-		logger.Error("Could not retrieve RBAC validator from context. [error]: %v", err)
+		aulogging.Logger.Ctx(ctx).Error().WithErr(err).Printf("Could not retrieve RBAC validator from context. [error]: %v", err)
```

Comment:
aulogging.Logger.Ctx is a pretty repetitive pattern. It feels like a bit of a "Java" style. Can we make this look a bit nicer?

Reply:
Oh, that's right, we agreed on the short style. We even had an issue for it :). Completely forgot.

Don't have a function for NoCtx() yet, but this is much rarer, and it's planned anyway.